src/oop/oop2.py

```diff
@@ -18,6 +19,12 @@ def __init__(self, num_wheels):
 # Override the drive() method in Motorcycle so that it returns "BRAAAP!!"
 
 # TODO
+class Motorcycle(GroundVehicle):
+    def __init__(self, num_wheels=2):
```

Comment:
this will allow `num_wheels` to be set to anything (but default to 2)..can you guarantee that `num_wheels` is set to 2?